src/core/data-tests.js

```diff
@@ -23,11 +23,34 @@ export const name = "core/data-tests";
 const lang = defaultLang in l10n ? defaultLang : "en";
 
 function toListItem(href) {
-  return hyperHTML.bind(document.createElement("li"))`
+  
+  let modifiedHref = href.split("/").pop().split(".html")[0];
+  
+  let emojiList = document.createElement('i');
+  
+  let requiresConnectionEmoji = hyperHTML.bind(document.createElement("i"))` 🔒 `;
+    requiresConnectionEmoji.setAttribute('aria-label','require-secure-connection');
```

Comment:
Note that aria-label is read out by a screen reader, so we want to make this as friendly as possible. I would suggest instead: "requires a secure connection".

We have quite a few users with visual impairments, so we do our best to support their needs.

Reply:
I really appreciate that :clap: